src/app/courses/courses.component.html

```diff
@@ -1,32 +1,97 @@
 <div class="mat-elevation-z8 courses">
   <p>{{message}}</p>
```

Comment:
We should move this, or possibly remove it, since the margin of the `p` tag is adding extra space at the top.